tests/projects/project1/packages/build.rs

```diff
@@ -0,0 +1,9 @@
+use std::process::{exit, Command};
+
+fn main() -> Result<(), std::io::Error> {
+    let ret = Command::new("buildsys").arg("build-package").status()?;
+    if !ret.success() {
+        exit(1);
+    }
+    Ok(())
+}
```

Comment:
I don't love the idea because someone might want to augment this code for their own purposes (pulling from a cache or generating some code, for example). On the other hand, without controlling this code, it is hard to update the calling pattern without breaking everyone.

One idea might be to vend a crates.io project that simplifies this down to:

```rust
fn main() -> Result<(), Error> {
    buildsys::run()
}
```

This would also eliminate the need vend buildsys as a binary. But, there would be a strict calling convention based on environment variables and some sort of version check to make sure the customer isn't using a too-old version of buildsys for Twoliter's expectations.